app/scripts/controllers/transactions.js

```diff
@@ -452,6 +462,7 @@ module.exports = class TransactionController extends EventEmitter {
       if (isKnownTx) return
       // encountered real error - transition to error state
       this.setTxStatusFailed(txMeta.id, {
+        stack: err.stack || err.message,
```

Comment:
so actually we dont use `stack:` any more, right? bc we generate our own stack trace?